rotkehlchen/constants/assets.py

```diff
@@ -2384,7 +2396,7 @@
     symbol='FOX',
     started=Timestamp(1553639047),
     swapped_for=None,
-    coingecko='fox-token',
+    coingecko='shapeshift-fox-token',
```

Comment:
oh good reminder. Whenever we change the globalDB we should also be recreating this file, to catch any such changes.